base/linalg/uniformscaling.jl

```diff
@@ -178,7 +178,7 @@ function copy!(A::AbstractMatrix, J::UniformScaling)
     return A
 end
 
-cond{T}(J::UniformScaling{T}) = J.λ ≠ zero(T) ? one(T) : convert(T, Inf)
+cond{T}(J::UniformScaling{T}) = J.λ ≠ zero(T) ? one(real(J.λ)) : oftype(real(J.λ), Inf)
```

Comment:
No, it should be: `one(real(J.λ)) : oftype(one(real(J.λ)), Inf)`

This is assuming that `one` is fixed to be dimensionless (since it is documented to be the multiplicative identity), as discussed in #16116.